doc/denite.txt

```diff
@@ -1451,12 +1451,64 @@ abbr		(String)			(Optional)
 ==============================================================================
 CREATE KIND						*denite-create-kind*
 
-Note: Not implemented yet...
+Custom Kind can be created to contain extra actions for your custom source.
+Before create your new kind, you should have a look at existing kinds in
+rplugin/python3/denite/kind/*.py
+
+Instead of create new kind from scratch, you can extend one of the existing
+Kind class, let's take `openable` for example: >
+
+	from ..kind.openable import Kind as Openable 
+
+	class Kind(Openable):
+		def __init__(self, vim):
```

Comment:
I noticed that an example code in `:h python` use a tab char..... I really don't like to use tab char for python that but if you prefer to use tab, I'm ok now.